images/cdi-artifact/werf.inc.yaml

```
@@ -5,7 +5,7 @@
 ---
 image: {{ $.ImageName }}
 final: false
-fromImage: builder/golang-bookworm-1.23
+fromImage: {{ eq $.SVACE_ENABLED "false" | ternary "builder/golang-bookworm-1.23" "builder/alt-go-svace" }}
```

Negation is always hard to follow, here is a straightforward expression:

```suggestion
fromImage: {{ ternary "builder/alt-go-svace" "builder/golang-bookworm-1.23" $.SVACE_ENABLED }}
```